Dockerfile

```diff
@@ -8,7 +8,6 @@ COPY opsdroid opsdroid
 COPY setup.py setup.py
 COPY versioneer.py versioneer.py
 COPY setup.cfg setup.cfg
-COPY requirements.txt requirements.txt
 COPY README.md README.md
 COPY MANIFEST.in MANIFEST.in
 
```

Comment:
I assume we want the docker file to install all the deps? If we do line 17 below needs updating to be `.[all]`

Reply:
Yeah the docker image should have everything. Maybe we should also have a slim image which just installs the core too.